sktime/forecasting/ttm.py

```diff
@@ -185,10 +186,10 @@ class TinyTimeMixerForecaster(_BaseGlobalForecaster):
     >>> from sktime.forecasting.ttm import TinyTimeMixerForecaster
     >>> from sktime.datasets import load_airline
     >>> y = load_airline()
-    >>> forecaster = TinyTimeMixerForecaster() # doctest: +SKIP
+    >>> forecaster = TinyTimeMixerForecaster()
     >>> # performs zero-shot forecasting, as default config (unchanged) is used
-    >>> forecaster.fit(y, fh=[1, 2, 3]) # doctest: +SKIP
-    >>> y_pred = forecaster.predict() # doctest: +SKIP
+    >>> _ = forecaster.fit(y, fh=[1, 2, 3])
```

Comment:
instead of doing this (which would confuse users), can you add a line with `TinyTimeMixerForecaster(...)` after the `fit`? That will catch anything that is printed between brackets and count as passed, if the dict does not get printed.